apps/src/templates/teacherHomepage/StudentHomepage.jsx

```diff
@@ -1,20 +1,44 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import HeadingBanner from '../HeadingBanner';
 import RecentCoursesCollapsible from './RecentCoursesCollapsible';
 import CollapsibleSection from './CollapsibleSection';
 import ResourceCard from './ResourceCard';
 import shapes from './shapes';
+import ProtectedStatefulDiv from '../ProtectedStatefulDiv';
 import i18n from "@cdo/locale";
 
+const styles = {
+  userHero: {
+    paddingTop: 10
+  }
+};
+
 const StudentHomepage = React.createClass({
   propTypes: {
     courses: shapes.courses
   },
 
+  componentDidMount() {
+    // The component used here is implemented in legacy HAML/CSS rather than React.
+    $('#user_hero').appendTo(ReactDOM.findDOMNode(this.refs.userHero)).show();
+  },
```

Comment:
I know we'd previously decided to render this partial in the DOM before any React content, but I think it's worth pulling into the React DOM now so that we can make use of the new `HeadingBanner` component on the page.